@xen-orchestra/lite/src/components/vm/VmsActionsBar.vue

```diff
@@ -2,7 +2,7 @@
   <MenuList
     :disabled="selectedRefs.length === 0"
     :horizontal="!isMobile"
-    :border="isMobile"
+    :no-border="isMobile"
```

Comment:
You need to inverse logic with `isMobile`
```suggestion
    :no-border="!isMobile"
```